alias/main.go

```diff
@@ -0,0 +1,59 @@
+package main
+
+import (
+	"fmt"
+	"os"
+	"os/exec"
+	"strings"
+)
+
+const GO_FLAG_INIT_STATE = "unset"
+
+// Go flag for the name of this binary on the filesystem
+var BinaryName = GO_FLAG_INIT_STATE
+
+// Shared token between this binary and sfdx. This enables us to know if sfdx was invoked by this process.
+var CliToken = GO_FLAG_INIT_STATE
+
+func main() {
+	var path string
+	var lookPathErr error
+	args := os.Args
+	args = append(args, CliToken)
+
+	// Check abs path locations
+	curPath := strings.Split(args[0], string(os.PathSeparator))
```

Comment:
args[0] isn't reliable, you should use something like this https://github.com/heroku/cli/blob/5329fb825c07d9956e600846c0306e5f98ff1497/filesystem.go#L48